docs: add linkedin meta tags, description to ai agents (#9835)
dagger/dagger: docs/current_docs/agents.mdx

---
slug: /ai-agents

> description: Learn how to build reproducible AI agents with Dagger using OpenAI, Anthropic, Gemini, or Ollama through containerized modules.

Reviewer: ```suggestion
description: Learn how to build reproducible AI agents with Dagger using OpenAI, Anthropic, Gemini, Llama, DeepSeek, Qwen and more.
```
